src/index.ts

```diff
@@ -113,21 +118,21 @@ const parseLoggerData = (data: string | AckeeLoggerOptions = {}) => {
     return { loggerName, options };
 };
 
-const loggerFactory = (data: string | AckeeLoggerOptions = {}): AckeeLogger => {
+const loggerFactory = (data: string | AckeeLoggerOptions = {}, loggerOptions: AckeeLoggerOptions = {}): AckeeLogger => {
+    // console.log('Factory called');
     const { loggerName, options } = parseLoggerData(data);
+    loggerOptions = objEmpty(options) ? loggerOptions : options;
+    const logger = defaultLogger(Object.assign({ loggerName }, loggerOptions));
 
-    if (!rootLogger) {
-        rootLogger = defaultLogger(options);
-        rootOptions = options;
-    }
-    if (!loggerName) {
-        return rootLogger;
-    }
-    return defaultLogger(Object.assign({ loggerName }, rootOptions));
+    const loggerProxy = makeCallable(logger, (childName: string) => {
+        const childLoggerName = [loggerName, childName].join('');
+        // console.log('Creating child', childName);
```

Comment:
:hankey: 

either
 - take advantage of NODE_DEBUG https://nodejs.org/docs/latest-v12.x/api/util.html#util_util_debuglog_section and keep the logs
 - add comment why is important to keep the code commented
 - remove the commented code

Reply:
Removed